docs/datasets-panel.rst

```diff
@@ -0,0 +1,104 @@
+==============
+Datasets panel
+==============
+Dataset panel is the central place to manage your datasets. Datasets are available to all sessions of a user. A user could choose to share dataset with other users.
+
+
+Upload
+======
+.. image:: img/datasets_panel-upload_button.png
+
+This is the initial state of dataset panel.
+You could use this upload button to upload dataset. See :doc:`/getting-started` for more detail.
+
+|
+|
+
+Dataset actions
+===============
+
+Visualize dataset
+-----------------
+.. image:: img/datasets_panel-globe_button.png
+
+The globe button after the dataset name adds the dataset to layer panel and visualize the dataset.
+
+Once the dataset is visualized, the globe button will be replaced by a remove button. Click this button to remove the dataset from visualization.
```

Comment:
Click this button to remove the dataset from visualization --> Click on this button to remove the dataset from the map